src/main/java/com/epam/esm/entity/Identifiable.java

```diff
@@ -0,0 +1,4 @@
+package com.epam.esm.entity;
+
+public interface Identifiable {
```

Comment:
Why do we need this interface?

Reply:
To mark entities and for the Dao interface